migration/sql-files/060-fixed-identities-username-idx.sql

```diff
@@ -0,0 +1,4 @@
+-- drop existing unique index
+DROP INDEX idx_idenities_username;
+-- recreate unique index idx_identities_username case insensitive username
```

Comment:
yep, this comment got in by mistake :-/. We don't forbid usernames such as `HectoR@redhat.com`, so we should lowercase the username in our queries.